Classes/GlobalStateExplorers/KeyChain/FLEXKeychain.m

```diff
@@ -0,0 +1,130 @@
+//
+//  SSKeychain.m
+//  SSKeychain
+//
+//  Created by Sam Soffes on 5/19/10.
+//  Copyright (c) 2010-2014 Sam Soffes. All rights reserved.
+//
+
+#import "FLEXKeychain.h"
+#import "FLEXKeychainQuery.h"
+
+NSString *const kSSKeychainErrorDomain = @"com.samsoffes.sskeychain";
```

Comment:
It looks like this is from
https://github.com/samsoffes/sskeychain/blob/master/SSKeychain.m
have you checked any copyright policy?

Reply:
Yes, it is from SSKeyChain, and it is [MIT LICENSE](https://github.com/samsoffes/sskeychain/blob/master/LICENSE) and i just renamed the file name and Classes. Should i do something more about the license?

Reply:
I will add the license